R-Interface/jasprcpp.cpp

```diff
@@ -114,7 +120,7 @@ void STDCALL jaspRCPP_init(const char* buildYear, const char* version, RBridgeCa
 	runCallbackCB								= callbacks->runCallbackCB;
 	readDataSetCB								= callbacks->readDataSetCB;
 
-
+	// TODO: none of this should pollute the global environment.
```

Comment:
Would it be better to manually make a sub-environment?

Reply:
yes, or we assign it into (a subenvironment of) the jaspBase or jaspResults namespace.